derive/src/lib.rs

```diff
@@ -430,6 +430,10 @@ fn process_metastructure_impl(s: synstructure::Structure, t: Trait) -> TokenStre
         binding.style = synstructure::BindStyle::MoveMut;
     }
     let mut from_value_body = TokenStream::new();
+    (quote! {
+        use lazy_static::lazy_static;
+        use regex::Regex;
```

Comment:
Can't we just qualify them in their usage?

Reply:
What do you mean by that?

Reply:
Basically to remove this block and use `regex::Regex` directly.

Reply:
Basically to remove this block and use `regex::Regex` directly.

Reply:
Ah sure